packages/lib/bookingSuccessRedirect.ts

```diff
@@ -179,14 +183,22 @@ export const useBookingSuccessRedirect = () => {
         navigateInTopWindow(url.toString());
         return;
       }
+
+      if (booking.uid) {
+        url.searchParams.append("uid", booking.uid);
+      }
+
       const bookingExtraParams = getBookingRedirectExtraParams(booking);
```

Comment:
Can we not make it work by just modifying `getBookingRedirectExtraParams` fn. That is the purpose of the function